.github/workflows/build_all.yml

```diff
@@ -665,7 +666,7 @@ jobs:
           SLACK_FOOTER: ' '
   Publish_to_firebase:
     name: Publish to firebase
-    if: github.event_name == 'push'
+    if: always() && github.event_name == 'push'
```

Comment:
don't use always here

Reply:
I need it to wait for the AGW job. But always work regardless of exit state.